tests/e2e/scenarios/scalability/run-test.sh

```diff
@@ -96,7 +98,8 @@ create_args+=("--node-count=${KUBE_NODE_COUNT:-101}")
 # However, it currently fails two tests (HostPort & OIDC) so need to track that down
 #create_args="--dns none"
 create_args+=("--node-size=c6g.medium")
-create_args+=("--master-size=c6g.2xlarge")
+create_args+=("--control-plane-count=2")
```

Comment:
It should be either 1 or 3. Also, not sure what the reason is to increase the control-plane number. It will probably decrease performance.
```suggestion
create_args+=("--control-plane-count=3")
```

Reply:
Thanks I was just trying to figure this out, should kops have this validation to fail the cluster creation if the size is 2?

Reply:
Also, the reason we want to add more CP nodes is to be able to perform a 5k kube nodes scale test. I have made this count configurable.

Reply:
2 CP nodes should also be doable, never tried it though. Let's do that separately.
Each CP has etcd running. The more etcd replicas, the slower it gets. (my reasoning)